terraform/example/main.tf

```diff
@@ -9,30 +13,90 @@ terraform {
 
 provider "teleport" {
     addr = "localhost:3025"
-    cert_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.crt"
-    key_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.key"
-    root_ca_path = "/Users/xnutsive/go/src/github.com/gravitational/teleport-plugins/docker/tmp/auth.cas"
+    cert_path = var.cert_path
+    key_path = var.key_path
+    root_ca_path = var.root_ca_path
 }
 
-resource "teleport_user" "tf_test" {
+resource "teleport_role" "tf_test" {
+  # kind = "testrolekind"
```

Comment:
These comments are not leftover. This lines are commented because I am waiting for validations to be implemented on Teleport side. They are needed, but specifying wrong values would break the Teleport instance. Let me add TODO comment here instead.

Reply:
That issue has been fixed on master, so you shouldn't need this anymore.

Reply:
Great. Comments removed.